src/dsp_tools/utils/excel_to_json_lists.py

```diff
@@ -335,7 +335,7 @@ def excel2lists(
     excel_file_paths = _extract_excel_file_paths(excelfolder)
     if verbose:
         print("The following Excel files will be processed:")
-        [print(f" - {filename}") for filename in excel_file_paths]
+        print(*(f" - {filename}" for filename in excel_file_paths), sep="\n")
```

Comment:
```suggestion
        print("\n".join((f" - {filename}" for filename in excel_file_paths)))
```
seems more readable to me